src/test/java/dev/jbang/BaseTest.java

```diff
@@ -61,6 +65,19 @@ void initEnv(@TempDir Path tempPath) throws IOException {
 
 	@BeforeAll
 	static void init() throws URISyntaxException, IOException {
+		try {
+			// The default ConsoleHandler for logging doesn't like us changing
```

Comment:
yeah so all this is why i refrained from using JUL for jbang output. why is this in the test class? shouldn't it be in main code path too?

Reply:
But in this module JUL is only used for logging, there is no essential output being generated with it. It's a library, it shouldn't do any essential output by itself, it only returns information which the caller can then decide to output or not. The logging is really only used for logging.
Thing is we do have tests that specifically check for certain log lines, we shouldn't really be doing that but I didn't want to change the tests so I added this code to make sure that the output between monolithic JBang and modular JBang would remain 99.9% the same.